index.js

```diff
@@ -19,8 +19,29 @@ function fastifyNext (fastify, options, next) {
     .then(() => {
       fastify
         .decorate('next', route.bind(fastify))
+        .decorateReply('nextRender', null)
+        .addHook('onRequest', (request, reply, done) => {
+          reply.nextRender = path => {
+            assert(typeof path === 'string', 'path must be a string')
+
+            for (const [headerName, headerValue] of Object.entries(reply.getHeaders())) {
+              if (headerName === 'content-length' && headerValue === undefined) {
```

Comment:
something about the error handler sets `content-length` to `undefined` - not sure why... _Might_ be a bug in fastify?

Tests fail without this condition

Reply:
we do not use `delete` because it will break V8 optimizations.

We remove the `content-length` because it might contain the length of the previous response.

Reply:
aha! so keeping the `continue` here makes sense?

Reply:
Or should fastify set it to an empty string instead of undefined?

Reply:
This code is correct as-is